src/components/TimeseriesChart/interfaces.ts

```diff
@@ -74,6 +74,10 @@ interface TimeseriesChartPropsBase {
    * Width of the chart
    */
   width?: number;
+  /**
+   * y domain displaying range (max minimum y values) configurations
```

Comment:
Description tells nothing for person, who would like to use this prop. It needs to be changed